readme.txt

```diff
@@ -449,7 +449,21 @@ Contributions are always welcome! Learn more about how to get involved in the [C
 
 = 3.0.0 =
 
-Starting with this release, modules such as Image Placeholders and Modern Image Formats are only available as standalone plugins as opposed to bundled modules. After updating, you will be able to easily migrate to this new structure.
+**Enhancements**
```

Comment:
Changelog added in `Upgrade Notice` instead of `Changelog` section

Reply:
Good catch. Fixed in 4bdbd0aac26fdc0a5297c72ab344cef6fc906b2e. This is something that needs to be fixed in `npm run readme`.